pkg/apis/serving/v1alpha1/revision_defaults.go

```diff
@@ -16,15 +16,9 @@ limitations under the License.
 
 package v1alpha1
 
-import (
-	"time"
-
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-)
-
 const (
 	// defaultTimeout will be set if timeoutSeconds not specified.
-	defaultTimeout = 60 * time.Second
+	defaultTimeout = 60
```

Comment:
Should this be qualified with `seconds`? Or even left as `time.Duration` and converted when needed?

Reply:
Done